.github/workflows/main-api.yml

```
@@ -0,0 +1,23 @@
+
+jobs:
+    test:
+        name: ?????
+        uses: ./.github/workflows/ci.yml
+
+    deploy:
+        name: ????
+        needs: test
+        uses: ./.github/workflows/deploy.yml
+        secrets:
+          HOST: ${{ secrets.MAIN_API_ADDRESS }}
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 10

(indentation)